src/dropin-model.js

```diff
@@ -214,7 +214,7 @@ function isPaymentOptionEnabled(paymentOption, options) {
   } else if (paymentOption === 'paypalCredit') {
     return gatewayConfiguration.paypalEnabled && Boolean(options.merchantConfiguration.paypalCredit);
   } else if (paymentOption === 'applePay') {
-    applePayEnabled = gatewayConfiguration.applePay && Boolean(options.merchantConfiguration.applePay);
+    applePayEnabled = gatewayConfiguration.applePay && Boolean(options.merchantConfiguration.applePayWeb);
```

Comment:
I think the change is backwards -- in my testing, it was `gatewayConfiguration` that had `applePayWeb`, and I think `merchantConfiguration` did have `applePay`.

Reply:
You're right, typo on my part. Fixing.